Random test failures in travis

Not had one of these in a while, but I think it would be a good idea to have one again. I'm noticing random test failures have picked up a bit again, and that usually means there's something not-so-random about them - a timing error or race condition.

If you restart a failed build, please log the error in a comment!
## Example

Job: https://travis-ci.org/TryGhost/Ghost/jobs/163983522
Error:

```
  1) Users API "before each" hook: setup for "Author CANNOT edit all roles":
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
```

This pops up more often now.

```
  1) Import Sanitizes removes duplicate tags and updates associations:
      AssertionError: expected 2 to be 1
      + expected - actual
      -2
      +1
      
    at Assertion.fail (node_modules/should/cjs/should.js:258:17)
    at Assertion.Object.defineProperty.value (node_modules/should/cjs/should.js:335:19)
    at core/test/integration/data/importer/importers/data_spec.js:79:54
```

Just got this one:

```
1) Import DataImporter imports data from 000:
      AssertionError: expected 'scheduled' to equal 'published'
      + expected - actual
      -scheduled
      +published
      
    at Assertion.fail (node_modules/should/cjs/should.js:258:17)
    at Assertion.Object.defineProperty.value (node_modules/should/cjs/should.js:335:19)
    at core/test/integration/data/importer/importers/data_spec.js:154:40
    at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (node_modules/bluebird/js/release/promise.js:638:18)
    at PromiseArray._resolve (node_modules/bluebird/js/release/promise_array.js:126:19)
    at PromiseArray._promiseFulfilled (node_modules/bluebird/js/release/promise_array.js:144:14)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:574:26)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
```

https://travis-ci.org/TryGhost/Ghost/jobs/262261787

--- a/core/test/integration/data/importer/importers/data_spec.js
+++ b/core/test/integration/data/importer/importers/data_spec.js
@@ -128,7 +128,7 @@ describe('Import', function () {
                 // Grab the data from tables
                 return Promise.all([
                     knex('users').select(),
-                    knex('posts').select(),
+                    models.Post.findPage(testUtils.context.internal),
                     knex('settings').select(),
                     knex('tags').select(),
                     knex('subscribers').select()
@@ -139,7 +139,7 @@ describe('Import', function () {
                 importedData.length.should.equal(5, 'Did not get data successfully');
 
                 var users = importedData[0],
-                    posts = importedData[1],
+                    posts = importedData[1].posts,
                     settings = importedData[2],
                     tags = importedData[3],
                     subscribers = importedData[4];
@@ -151,8 +151,8 @@ describe('Import', function () {
 
                 // import no longer requires all data to be dropped, and adds posts
                 posts.length.should.equal(exportData.data.posts.length, 'Wrong number of posts');
-                posts[0].status.should.eql('published');
-                posts[1].status.should.eql('scheduled');
+                posts[0].status.should.eql('scheduled');
+                posts[1].status.should.eql('published');
 
                 // test settings
                 settings.length.should.be.above(0, 'Wrong number of settings');

